packages/cerebral/src/Compute.js

```diff
@@ -0,0 +1,30 @@
+import Tag from './tags/Tag'
+
+export default class Compute {
+  constructor(args) {
+    this.args = args
+    this.value = null
+  }
+  getValue(get) {
+    const result = this.args.reduce((details, arg, index) => {
+      if (arg instanceof Compute || arg instanceof Tag) {
+        details.results.push(arg.getValue(get))
+
+        return details
+      } else if (typeof arg === 'function') {
+        details.results.push(arg(...details.results.slice(details.previousFuncIndex, index), get))
```

Comment:
shouldn't `previousFuncIndex` be altered here `details.previousFuncIndex = index` ?